core/templates/dev/head/domain/statistics/PlaythroughImprovementTaskObjectFactorySpec.ts

```diff
@@ -328,16 +328,6 @@ describe('PlaythroughImprovementTaskObjectFactory', function() {
       });
     });
 
-    describe('.getActionButtons', function() {
```

Comment:
Shouldn't this test be replaced by an equivalent one (maybe after mocking a URL fetch for the user info)?

Reply:
Done.